source/arm9/hw_math.c

```diff
@@ -0,0 +1,35 @@
+
+
+#include <math.h>
+
+
+
+f32 hw_sqrtf(f32 x){
```

Comment:
`f32` is a libnds type, right? I think using `float` makes more sense, in order to make it clear that it's a simple float. Using types like `int32_t` and such is fine, probably.

Also, the bracket needs to go in the next line.